content/docs/apps/custom-domains.md

```diff
@@ -109,7 +112,10 @@ cloud.gov offers a managed service that allows you to forward a domain you contr
     Message: Provisioning in progress; CNAME domain "my.domain.gov" to "d3kajwa62y9xrp.cloudfront.net."
     ```
 
+1. Repeat last step until the CloudFront distribution is provisioned. Note this can take up to 30 minutes.
 1. Create/update your DNS configuration.
-1. Wait up to 30 minutes for the CloudFront distribution to be provisioned and the DNS changes to propagate.
+    * In AWS Route 53, this will (confusingly) be an `A` record with an `Alias` to `<something>.cloudfront.net`.
+    * If you can, wait to **do this step during low-traffic hours**. It can take up to 15 minutes before the certificate is in place, and visitors will get a certificate error when visiting the custom domain in the meantime.
```

Comment:
Users have to update DNS records before the acme challenge--to pass the challenge, we need to prove that we control the domain. So I'm not sure how to change domains without downtime.